kai/service/llm_interfacing/model_provider.py

```diff
@@ -51,9 +52,16 @@ def __init__(self, config: KaiConfigModels):
 
             case "ChatIBMGenAI":
                 model_class = LangChainChatInterface
-
+                if os.getenv("DEMO_MODE") == "true":
```

Comment:
We need to account for various forms of case with the value "true", as this could be "True", "TRUE", etc...
For example:  https://github.com/konveyor/kai/blob/main/compose.yaml#L4

we could consider something like:
`if os.getenv("DEMO_MODE", "false").lower() == "true":`

Reply:
Thanks for pointing that out. I’ve updated the code to handle various cases of the "true" .